### PR TITLE
Adding a function to get an author's email address.

### PR DESCRIPTION
**Pull request by bquade-cpr**
Excluding byline manager profiles for now because I don't think they have an email address and I don't know how to test it.

### CODE REVIEW

File: inc/wp-components/traits/trait-author.php [hunk 1]
```diff
@@ -87,6 +87,33 @@ public function get_author_display_name() : string {
...
+
+			case 'guest_author':
+				$email_address = $this->guest_author->user_email;
+				break;
+
+			/*
+			case 'byline_manager_profile':
+				$email_address = $this->byline_manager_profile->user_email;
+				break;
+			*/
```
Comment (CPROps) [inline inc/wp-components/traits/trait-author.php]: We should remove this commented chunk before going live.

File: inc/wp-components/traits/trait-author.php [hunk 2]
```diff
@@ -87,6 +87,33 @@ public function get_author_display_name() : string {
...
+		$email_address = '';
+
+		switch ( $this->get_author_type() ) {
+			case 'wp_user':
+				$email_address = $this->user->data->user_email;
+				break;
+
+			case 'guest_author':
+				$email_address = $this->guest_author->user_email;
+				break;
```
Comment (CPROps) [inline inc/wp-components/traits/trait-author.php:105]: Not a blocker - but you might want a default case, here - for other roles to break on.

**Review by CPROps — CHANGES_REQUESTED** [review]
Two suggestions - one is not blocking, just a recommendation. 
Getting rid of the commented-out code will also fix the Travis build error. 

Other than that, looks/functions great. Thanks!

**Review by slothluvchunk — APPROVED** [review]

**Review by CPROps — APPROVED** [review]
🐴

**Comment by CPROps** [comment]
PR approved, merging.

**Pull request merged**